tests/opentelemetry-docker-tests/tests/pymysql/test_pymysql_functional.py

```diff
@@ -84,23 +84,23 @@ def test_execute(self):
         stmt = "CREATE TABLE IF NOT EXISTS test (id INT)"
         with self._tracer.start_as_current_span("rootSpan"):
             self._cursor.execute(stmt)
-        self.validate_spans("CREATE")
+        self.validate_spans("CREATE test")
```

Comment:
@keithZmudzinski, they are legit tests that need to be fixed if we want this PR to be merged.